mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/openjpa/JPAMailboxMessage.java

```diff
@@ -49,6 +49,7 @@
 public class JPAMailboxMessage extends AbstractJPAMailboxMessage {
 
     private static final byte[] EMPTY_BODY = new byte[] {};
+    private static final byte[] EMPTY_HEADER = new byte[] {};
```

Comment:
Maybe we just need one of these `EMPTY_ARRAY` constant, returned in both cases?

Reply:
Pushed change.